sdks/python/apache_beam/runners/worker/operations.pxd

```diff
@@ -62,24 +63,24 @@ cdef class Operation(object):
   cpdef start(self)
   cpdef process(self, WindowedValue windowed_value)
   cpdef finish(self)
-
   cpdef output(self, WindowedValue windowed_value, int output_index=*)
+  cpdef progress_metrics(self)
+
 
 cdef class ReadOperation(Operation):
   @cython.locals(windowed_value=WindowedValue)
-  cpdef start(self)
```

Comment:
We need to keep this since the line above decorates it. Not sure why that decoration is needed though.

Reply:
This types the local variable: http://cython.readthedocs.io/en/latest/src/tutorial/pure.html#augmenting-pxd